django/contrib/admin/static/admin/js/popup_response.js

```diff
@@ -3,6 +3,9 @@
     'use strict';
     var initData = JSON.parse(document.getElementById('django-admin-popup-response-constants').dataset.popupResponse);
     switch(initData.action) {
+    case 'add':
```

Comment:
What are the steps to reproduce this? I'm not seeing the "Yes, I'm sure" button on the page that says protected objects cannot be deleted.